DuckDuckGo.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -32,17 +32,17 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/duckduckgo/BrowserServicesKit",
       "state" : {
-        "revision" : "20316b105d8874e10cf544a9c97e97237441ae01",
-        "version" : "224.6.0"
+        "branch" : "sam/add-persistence-dependency-to-crashes-module",
+        "revision" : "93e72b565f5331a13deac01b944a5087352fd908"
       }
     },
     {
       "identity" : "content-scope-scripts",
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/duckduckgo/content-scope-scripts",
       "state" : {
-        "revision" : "06d244c4e0951ef16217c46f23c4865c5f98dd5c",
-        "version" : "7.4.0"
+        "revision" : "0502ed7de4130bd8705daebaca9aeb20d3e62d15",
+        "version" : "7.5.0"
```

Comment:
Looks like some of our other dependencies were out of date in Package.resolved, I'm including those bumps here.